index.d.ts

```diff
@@ -45,11 +45,24 @@ declare namespace unusedFilename {
 	}
 
 	/**
-	 * An error thrown when maxTries limit has been reached without finding an unused path.
-	 *
-	 * @param originalPath - Path without the incrementation sequence.
-	 * @param lastTriedPath - Last tested incremented path.
-	 */
+	An error thrown when maxTries limit has been reached without finding an unused path.
+
+	@param originalPath - Path without the incrementation sequence.
+	@param lastTriedPath - Last tested incremented path.
+
+	@example
+	```
+	const MaxTryError = unusedFilename.MaxTryError;
```

Comment:
Forgot to mention this earlier, but I tend to include import statement in all code examples, so the examples can be used directly.